CONTRIBUTING.md

```diff
@@ -0,0 +1,89 @@
+# Contributing to the Rails Girls Summer of Code website
+
+We're always very happy about contributions to our website, so thank you for taking the time to contribute! Here you will find some guidelines about what is expected from contributors. Before getting started, please make sure you read and understand our [Code of Conduct](https://github.com/rails-girls-summer-of-code/summer-of-code/blob/gh-pages/CODE_OF_CONDUCT.md), and be aware that every contributor is expected to abide by it.
+
+#### Table of contents
+
+1. [Setting up and getting started](#setting-up-and-getting-started)
+2. [Reporting a bug](#reporting-a-bug)
+3. [Suggesting a new feature](#suggesting-a-new-feature)
+4. [Contributing to an existing issue](#contributing-to-an-existing-issue)
+5. [Additional info](#additional-info)
+
+
+
+
+## Setting up and getting started
+
+The Rails Girls Summer of Code website is built using [Jekyll](https://jekyllrb.com/). In order to set up and run the site locally for development, you'll need to have Ruby installed on your machine, create a GitHub account (if you don't already have one), and fork our repo.  
+"Forking" basically just means "making a copy"; this allows you to copy our `summer-of-code` repo (found [here](https://github.com/rails-girls-summer-of-code/summer-of-code)) without affecting it. Follow the instructions on forking a repo and cloning your fork [in the GitHub docs](https://help.github.com/articles/fork-a-repo/). When forking, you might be asked where you want to fork the repository to; select **your own Github profile**.  
+Once you're all set up, you'll have to bundle the gems using bundler and start jekyll:
+
+```
+gem install bundler
+bundle install
+jekyll serve --watch
+
+```
+
+To see the site locally, just head to `http://localhost:4000` in your browser. You're ready to go!
```

Comment:
I'dd add another reference to the Jekyll docs, something like "If you are new to Jekyll you can find more about it [here](http://jekyllrb.com/docs/home/)"